symbols.js

```diff
@@ -395,12 +395,25 @@ const symbols = [
     }
 ];
 
+let symbols = []
+
+try {
+    // Catching invallid json, not an invalid symbol array
+    symbols = JSON.parse(window.localStorage.getItem("symbols"));
+} catch(err) {
+    console.error(err);
+    symbols = []
```

Comment:
```suggestion

```

I think this line is superfluous, `symbols` is set to the empty array on 398 already.

Reply:
OK, implemented in 914415c9dd6387c7983bcf6ddd23508b77b5c2fe